applications/doodle/src/doodle_inbound_listener.erl

```diff
@@ -177,6 +192,10 @@ terminate(_Reason, _State) ->
 code_change(_OldVsn, State, _Extra) ->
     {'ok', State}.
 
+format_status(_Opt, [_PDict, _State]) -> [].
+
+handle_debug(FuncState, _Event, _ProcState) ->  FuncState.
```

Comment:
how do we call this?
Do you plan to update all config files with the debug flag or will the config files in our environments be unchanged?